docs/how_to_write_packages.md

```diff
@@ -101,19 +102,19 @@ package pods
 
```

Comment:
I can't add a comment on the correct line, but above this snippet it says "rule in a package named `podsbestpractices`" but the snippet then declares `package pods`.

Reply:
Should be fixed in f6dc60a27601b88944a377648df345d29783b74f